src/main/java/com/app/notebook/controller/FileController.java

```diff
@@ -0,0 +1,34 @@
+package com.app.notebook.controller;
+
+import com.app.notebook.model.File;
+import com.app.notebook.model.dto.UploadFileDto;
+import com.app.notebook.service.FileService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+/**
+ * notebook-api Created by Catalin on 10/21/2020
+ */
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = "/api/v1/files")
+public class FileController {
+
+    private FileService fileService;
+
+    @Autowired
+    public FileController(FileService fileService) {
+        this.fileService = fileService;
+    }
+
+    @PostMapping(value = "/save")
```

Comment:
cred ca e suficient sa faci Post pe / ca sa faci save.. nu e nevoie sa denumesti /save -> bad practice sa pui verbu in path :))